### ISSUE TITLE
Admins are granted money for deleting other players' plots

### ISSUE THREAD

**Issue by MegaNarwhal** [IntellectualSites/PlotSquared#1770]
When deleting players' plots, you are granted the same amount of money as if you deleted your own plot. This is mostly an issue when using the confirmation mode for plot expiry, as you have to take money from yourself after clearing plots because you get paid every time you reset one, but it also manifests when claiming and merging a bunch of plots for a spawn or just resetting undesirable, non-expired plots. I see a few possible solutions:
- Either economy functions could be disabled completely when dealing with other users' plots
- A new option could be added to exempt admins from PlotSquared economy integration
- A new cost/reward config option could be added only for resetting OTHER users' plots, in addition to the current one for resetting any plot.

**Comment by Sauilitired** [comment]
> Either economy functions could be disabled completely when dealing with other users' plots
That should probably be it.

**Comment by MegaNarwhal** [comment]
Agreed, seems like the simplest way to do it and I can't think of any reasonable use cases that benefit from the way economy integration currently functions when interacting with other players' plots.

**Comment by Legoman99573** [comment]
Not bad to add, but would make it configurable as some owner see it as abusable if the amount is high. I would set it to add $1 per plot deletion by default to prevent abuse ;)
